ROMFS: add 13100_generic_tiltrotor config and remove actuator assignments from other generic vehicles

sfuhrer opened the pull request:
Following up on https://github.com/PX4/PX4-Autopilot/pull/19596#issuecomment-1119658490, I created a "Generic Tiltrotor VTOL" airframe config. The purpose of it is that users with custom airframes (=not RTF models basically) can select this airframe, it sets the needed system variables that tell the system that we're dealing with a VTOL tiltrotor (yes, we should try to reduce their number, they're quite redundant) and then the user configures the actuators through the control allocation UI. 

We already have generic airframes for tailsitters(13200) and standard VTOLs (13000), though they also contain specific actuator settings that we maybe want to remove after dropping support for the legacy mixing system. IMO VTOLs (and FW btw) are too diverse to split them further up than "standard VTOL", "tiltrotor" and "tailsitter", though we can talk about this.

dagar commented:
I'm not sure, this makes me think a reasonable baseline for an airframe existing (with a few exceptions) might be that it defines the actuators (CA_ROTOR, CA_SV, etc)? 🤷 

We could probably figure out a mechanism to eliminate `MAV_TYPE` and `VT_TYPE` after the control allocation switch over.

sfuhrer commented:
>it defines the actuators (CA_ROTOR, CA_SV, etc)?

But with which tiltrotor design would you go with? There are so many options: tricopter, quadcopter, only front motors tiltable, flying-wing design with elevons, normal AAER config, V-tail, A-tail, ... It's really straight forward to configure the actuators now, and in general I think it's better if a user setting up his custom vehicle starts with a clean sheet and adds all the actuator settings himself, instead of him needed to know which ones to alter for his setup.

sfuhrer commented:
That's how the setup of a random tiltrotor VTOL would look like - you select your type (Generic VTOL Tiltrotor) and they can start with allocating the actuators and follow the other "red tabs" in QGC to complete the configuration of the vehicle. 
I think there are the fewest amount of wrong airfraame setups if the user can start with a blank sheet for the actuators and fill them in one by one. The UI is clear enough, and we can easily document it to be as bulletproof as possible. 
![QGC_airframe_selection_CA](https://user-images.githubusercontent.com/26798987/173320078-3fc291a8-f26e-4e44-8e36-ffcd9bffd061.gif)

sfuhrer commented:
@dagar as discussed I added a default geometry for all the generic airframes. Thus the users will not start with a blank configuration tab but can change the actuator geometry as required. The output section needs to be setup from scratch. 
I also removed the legacy mixers from the generic airframes.
